classes/reportbuilder/local/systemreports/matching_users.php

```diff
@@ -48,16 +48,17 @@ protected function initialise(): void {
         $this->add_entity($userentity);
 
         $conditions = $rule->get_condition_records();
+        $sql = condition_manager::build_sql_data($conditions, $rule->get('operator'));
```

Comment:
Changes in this file are related to an error that was displayed when a rule doesn't have any conditions. Found that while working on new UI.